api/migrations/versions/1c3fda18bad6_.py

```diff
@@ -0,0 +1,413 @@
+    
+    about_you_q4 = Question(app_form.id, about_you.id, 'long-text', 'Share with us a favourite machine learning resource you use: a paper, blog post, algorithm, result, or finding. Tell us why.', 'Enter up to 80 words', 4, validation_regex = "^\s*(\S+(\s+|$)){0,80}$", description = 'Enter up to 80 words, remember to include *why*')
+    about_you_q4.depends_on_question_id = main_q1.id
+    about_you_q4.show_for_values = ['undergraduate, masters, phd, post-doc, student at a coding academy, unemployed, academic-faculty']
+
+    about_you_q5 = Question(app_form.id, about_you.id, 'long-text', 'Are you or have you been a tutor for any relevant course, or part of any machine learning or data science society or meetup? If yes, give details.', 'Enter up to 80 words', 5, validation_regex = '^\s*(\S+(\s+|$)){0,80}$', description = 'Enter up to 80 words')
+    about_you_q5.depends_on_question_id = main_q1.id
+    about_you_q5.show_for_values = ['undergraduate, masters, phd, post-doc, student at a coding academy, unemployed']
+
+    about_you_q6 = Question(app_form.id, about_you.id, 'long-text', 'Have you taught any Machine Learning courses at your institution or supervised any postgraduate students on Machine Learning projects?', 'Enter up to 80 words', 6, validation_regex = '^\s*(\S+(\s+|$)){0,80}$', description = 'Enter up to 80 words')
+    about_you_q6.depends_on_question_id = main_q1.id
+    about_you_q6.show_for_values = ['academic-faculty']
+
+    about_you_q7 = Question(app_form.id, about_you.id, 'multi-choice', 'Are you currently actively involved in Machine Learning Research?', 'Choose an option', 7, None, description = 'Choosing no will not count against you',
+                   options = [
+                      {'label': 'Yes', 'value': 'yes'},
+                      {'label': 'No', 'value': 'no'}
+                   ])
+    about_you_q8 = Question(app_form.id, about_you.id, 'long-text', 'Add a short abstract describing your current research', 'Enter 150 to 250 words', 8, validation_regex = '^\s*(\S+(\s+|$)){150,250}$', description = 'This can be completed research or research in progress. Remember to include a description of your methodology and any key results you have so far.')
+    about_you_q8.depends_on_question_id = about_you_q7.id
+    about_you_q8.show_for_values = ['yes']
+
+    about_you_q9 = Question(app_form.id, about_you.id, 'multi-choice', 'Would you be interested in submitting an extended abstract or paper to the Indaba Symposium if you are selected to attend the Indaba?', 'Choose an option', 9, None, description = "We won't hold you to this, it's just to gauge interest.")
+    about_you_q9.depends_on_question_id = about_you_q7.id
+    about_you_q9.show_for_values = ['yes']
+
+    about_you_q10 = Question(app_form.id, about_you.id, 'long-text', 'Have you worked on a project that uses machine learning? Give a short description.', 'Enter upto 150 words', 10, validation_regex = '^\s*(\S+(\s+|$)){0,150}$')
```

Comment:
Please make this one depend on question `about_you_q7` and show for values `['no']` (sorry, my mistake for forgetting to include it in the spec document)